gluon/tools.py

```diff
@@ -777,16 +777,16 @@ def encoded_or_raw(text):
                 if attachments:
                     result = mail.send_mail(
                         sender=sender, to=origTo,
-                        subject=unicode(subject), body=unicode(text), html=html,
+                        subject=str(subject), body=str(text), html=html,
```

Comment:
Great! I think it's better than matclab's solution.

Will send another commit to the PR.